src/components/Editor/GeneralEditor.test.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { GeneralEditor } from './GeneralEditor';
+import { VideoEditorToolbar } from './Editors/VideoEditorToolbar';
+import { JSDOM } from 'jsdom';
```

Comment:
there is no need to hook to jsdom directly. Jest uses jsdom under the hood

Reply:
done